src/scss/_tab-statements.scss

```diff
@@ -3,109 +3,91 @@
-    @include declaration('text-decoration', $tabsnavs--text-decoration);
-    @include declaration('border-width', $tabsnavs--border-width);
-    @include declaration('border-style', $tabsnavs--border-style);
-    @include declaration('border-color', $tabsnavs--border-color);
-    @include declaration('border-radius', $tabsnavs--border-radius);
-    @include declaration('color', $tabsnavs--color);
-    @include declaration('box-shadow', $tabsnavs--box-shadow);
-    @include declaration('background', $tabsnavs--background);
-    @include declaration('background-color', $tabsnavs--background-color);
-
-    @include block-generation($tabsnav--selector);
-  }
-
-
-  $tabsnav-item--selector: selector--set-element($tabsnav--selector, 'item');
-  $tabsnav-item-nextto-item--selector: selector--append-parent($tabsnav-item--selector, $tabsnav-item--selector, '+');
-
-  @include selector($tabsnav-item-nextto-item--selector) {
-    @include declaration('margin-left', $tabsnavs-item--gutter);
-
-    @include block-generation($tabsnav-item-nextto-item--selector);
-  }
-
-
-  $tabsnav-anchor--selector: selector--set-element($tabsnav--selector, 'anchor');
-
-  @include selector($tabsnav-anchor--selector) {
-    @include declaration('cursor', pointer);
-    @include declaration('cursor', default, '~active');
-    @include declaration('display', block);
-    @include declaration('font-size', $tabsnavs-anchor--font-size);
-    @include declaration('font-family', $tabsnavs-anchor--font-family);
-    @include declaration('font-weight', $tabsnavs-anchor--font-weight);
-    @include declaration('text-transform', $tabsnavs-anchor--text-transform);
-    @include declaration('letter-spacing', $tabsnavs-anchor--letter-spacing);
-    @include declaration('line-height', $tabsnavs-anchor--line-height);
-    @include declaration('text-align', $tabsnavs-anchor--text-align);
-    @include declaration('text-decoration', $tabsnavs-anchor--text-decoration);
-    @include declaration('margin', $tabsnavs-anchor--margin);
-    @include declaration('border-width', $tabsnavs-anchor--border-width);
-    @include declaration('border-style', $tabsnavs-anchor--border-style);
-    @include declaration('border-color', $tabsnavs-anchor--border-color);
-    @include declaration('border-radius', $tabsnavs-anchor--border-radius);
-    @include declaration('color', $tabsnavs-anchor--color);
-    @include declaration('box-shadow', $tabsnavs-anchor--box-shadow);
-    @include declaration('background', $tabsnavs-anchor--background);
-    @include declaration('background-color', $tabsnavs-anchor--background-color);
-    @include declaration('padding', $tabsnavs-anchor--padding);
-
-    @include block-generation($tabsnav-anchor--selector);
-  }
+  $tabsnav--selector: selector(new 'tabs-nav' with $collapses--syntax);
+
+  @include statement($tabsnav--selector, (
+    (padding, if($tabsnavs--padding, $tabsnavs--padding, 0)),
+    (margin, if($tabsnavs--margin,$tabsnavs--margin, 0 )),
```

Comment:
Add a space after the first comma of the function `if()` and remove the space after the third argument